.gitignore

```diff
@@ -85,6 +85,7 @@ celerybeat-schedule
 # virtualenv
 .venv
 venv/
+myvenv/
```

Comment:
1) You need to ignore SQLite3 DB.
2) Try switching to a pyenv-controlled virtualenv, it stores its folders separately.